test/runtimeconfig.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "crictl info prints out the crio runtime config" {
+	start_crio
+	info_json=$(crictl info -o json)
+
+	run -0 jq -e '.config.crio' <<< "$info_json"
+
+	default_runtime=$(jq -r ".config.crio.DefaultRuntime" <<< "$info_json")
+	expected=$(get_runtime)
+	[[ "${default_runtime}" == "$expected" ]]
+
+	run -0 jq -e ".config.crio.Runtimes.${default_runtime}" <<< "$info_json"
```

Comment:
https://bats-core.readthedocs.io/en/stable/writing-tests.html#when-not-to-use-run
```suggestion
	jq -e ".config.crio.Runtimes.${default_runtime}" <<< "$info_json"
```